web/custom-plugins/forecast-of-response/src/components/Comments.tsx

```diff
@@ -49,7 +71,81 @@ export const CompactCommentView = ({ comment }: ICompactCommentView) => {
       <div style={{ borderBottom: 'black 1px solid', width: 'fit-content' }}>
         {comment.operation}
       </div>
-      <MessageWrapper>{comment.message}</MessageWrapper>
+      <CompactMessageWrapper>{comment.message}</CompactMessageWrapper>
+    </CompactCommentWrapper>
+  )
+}
+
+export const CommentView = (props: { comment: TComment; even: boolean }) => {
+  const { comment, even } = props
+  const uniqueAuthorColor = colorFromString(comment.author)
+  return (
+    <CommentWrapper color={uniqueAuthorColor} even={even}>
+      <CommentHeaderWrapper>
+        <IconWrapper color={uniqueAuthorColor}>&#9679;</IconWrapper>
+        <b>{comment.author}</b>
+      </CommentHeaderWrapper>
+      <b style={{ width: 'fit-content' }}>
+        {new Date(comment.date).toLocaleString()}
```

Comment:
we can probably use the locale 'en-GB' here

Reply:
fixed!